test/puppetlabs/puppetdb/scf/migrate_test.clj

```diff
@@ -25,9 +25,11 @@
 (use-fixtures :each tdb/call-with-test-db)
 
 (defn apply-migration-for-testing!
-  [i]
+  [i & args]
   (let [migration (migrations i)
-        result (migration)]
+        result (if (empty? args)
+                 (migration)
+                 (apply migration args))]
```

Comment:
Think you may not need the `if` here -- `(apply migration args)` will do the right thing either way.

Reply:
Thought this blew up when `nil` was passed, but looks like it's doing the right thing. Changed.